spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+RSpec.describe Airbrake::Shoryuken::ErrorHandler do
+  let(:error) { AirbrakeTestError.new('shoryuken error') }
+  let(:body) { { message: 'message' } }
+  let(:notice) { double 'notice' }
+  let(:worker) { FooWorker.new }
+  let(:endpoint) do
+    'https://airbrake.io/api/v3/projects/113743/notices?key=fd04e13d806a90f96614ad8e529b2822'
+  end
+
+  class FooWorker; end
```

Comment:
To avoid defining a class in global scope (so we don't leak data from this test), I would use the following:

```rb
let(:worker) do
  Class.new do
    def to_s
      'FooWorker'
    end
  end.new
end
```

I know, looks unusual, but does the job :) If you have better ideas, I'm happy to see them, too.